app/controllers/guides_controller.rb

```diff
@@ -21,4 +21,12 @@ def edit
       redirect_to @guide
     end
   end
+
+  def destroy
+    @guide = current_user.guides.find(params[:id])
```

Comment:
You could do this through the mutation as well, that way we keep all of the code to do with the actual destruction in one place.